src/PositionManager.sol

```diff
@@ -217,8 +217,10 @@ contract PositionManager is
         bytes calldata hookData
     ) internal onlyValidConfig(tokenId, config) {
         // Note: The tokenId is used as the salt for this position, so every minted position has unique storage in the pool manager.
-        BalanceDelta liquidityDelta = _modifyLiquidity(config, liquidity.toInt256(), bytes32(tokenId), hookData);
-        liquidityDelta.validateMaxInNegative(amount0Max, amount1Max);
+        (BalanceDelta liquidityDelta, BalanceDelta feesAccrued) =
+            _modifyLiquidity(config, liquidity.toInt256(), bytes32(tokenId), hookData);
+        // Slippage checks should be done on the principle liquidityDelta which is the liquidityDelta - feesAccrued
```

Comment:
```suggestion
        // Slippage checks should be done on the principal delta which is the liquidityDelta - feesAccrued
```

for all occurrences

Reply:
principle vs principal - for a loan etc its principal

Reply:
principle vs principal is just something i will never get right i swear